UniWatch/ViewModel/EnrollViewModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UniWatch.Models;
+
+namespace UniWatch.ViewModels
+{
+    public class EnrollViewModel
```

Comment:
`EnrollViewModel` is never used. Is this class supposed to be `EnrollIndexViewModel` or `EnrollStudentViewModel`?